lib/miq_environment.rb

```diff
@@ -68,8 +68,9 @@ def self.is_container?
     def self.is_podified?
       return @is_podified unless @is_podified.nil?
 
-      require "container_orchestrator"
-      @is_podified = is_container? && ContainerOrchestrator.available?
+      @is_podified = is_container? &&
+                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/token") &&
+                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/ca.crt")
```

Comment:
Maybe tack in a code comment?

```suggestion
      # NOTE: The kubernetes files listed here are duplicated in ContainerOrchestrator.available?
      #   however we cannot load container_orchestrator.rb, as this method is called in a pre-boot
      #   environment.  For now, we are duplicating these file checks.
      @is_podified = is_container? &&
                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/token") &&
                     File.exist?("/run/secrets/kubernetes.io/serviceaccount/ca.crt")
```

Reply:
Can't we make these paths as CONSTANTS somewhere accessible by both?

Reply:
I can certainly try that, maybe I'm just weary at this point but I'd prefer to do that in a follow-up rather than re-build, deploy, and verify this whole thing

Reply:
Let's do this in a follow up....want to get us to a green build first.